src/panel.html

```diff
@@ -2,11 +2,20 @@
 <html style="height: 100%;">
   <head>
     <meta charset="utf-8" />
+    <style>
+      body {
+        background: transparent;
+        font-family: sans-serif;
+      }
+      body.dark {
+        background-color: #272822;
+        color: #f8f8f2;
+      }
+    </style>
+    <script></script>
   </head>
   <body style="margin: 0; width: 100%; height: 100%;">
-    <div id="app" style="width:100%; height: 100%">
-      Loading the single-spa inspector...
-    </div>
```

Comment:
Don't we lose the loading state here? Why would we want to lose this?

Reply:
I removed that one in favor of the one rendered by the app itself [panel-app.js#L37-L39](https://github.com/CanopyTax/single-spa-inspector/blob/master/src/panel-app.js#L37-L39). The two messages are different and the other one seems more valuable.

Reply:
:heart: